sig-release/annual-report-2022.md

```diff
@@ -2,76 +2,105 @@
 
 ## Current initiatives
 
-1. What work did the SIG do this year that should be highlighted?
+SIG Release has a [roadmap](https://github.com/kubernetes/sig-release/blob/master/roadmap.md) that captures high level initiatives that the SIG is working toward, with specific information captured in our [project board](https://github.com/orgs/kubernetes/projects/30). 
 
-   -
-   -
-   -
+1. What work did the SIG do this year that should be highlighted?
+   - [Migrated most of deb/rpm package building into release process to reduce Google Build Admin involvement in releases](https://github.com/kubernetes/release/issues/2737)
+   - Proof-of-concept of using OpenSUSE Build Service to build and publish packages using community infrastructure. Reflected in updates to [1731 - Publishing Kubernetes packages on community infrastructure](https://github.com/kubernetes/enhancements/tree/master/keps/sig-release/1731-publishing-packages). See [biweekly meeting from 18 October 2022](https://youtu.be/8l8X3vSAJAw?t=787) for a good overview.
+   - [Signing of Release Artifacts](https://github.com/kubernetes/enhancements/issues/3031).
 
 2. What initiatives are you working on that aren't being tracked in KEPs?
```

Comment:
I'm wondering if we should highlight here or in # 1 the work to keep the image promotion process working under the registry changes. Last year the promoter changes represented a big chunk of the engineering time we put into SIG Release during the second half of the year. We had a lot of releases that demanded a lot of time from the release managers, often spending several days promoting the images.

Reply:
added text about this.